src/sentry/models/dashboard.py

```diff
@@ -18,9 +17,6 @@ class Dashboard(Model):
     created_by = FlexibleForeignKey("sentry.User")
     organization = FlexibleForeignKey("sentry.Organization")
     date_added = models.DateTimeField(default=timezone.now)
-    status = BoundedPositiveIntegerField(
```

Comment:
You'll need to update tests, and likely the endpoints that read dashboards if they were doing conditions on this field before.